superset/views/core.py

```diff
@@ -2644,8 +2662,9 @@ def sql_json_exec(
             )
 
         # set LIMIT after template processing
-        limits = [mydb.db_engine_spec.get_limit_from_sql(rendered_query), limit]
-        query.limit = min(lim for lim in limits if lim is not None)
+        if not config.get("SQLLAB_CTA_NO_LIMIT", False):
```

Comment:
We can assume that the key exists no need for default to False here, and if would be False anyways if the key doesn't exist

Reply:
good point :)